docs/source/03_tutorial/07_version_metrics.md

```diff
@@ -1,3 +1,57 @@
 # Version metrics
 
-This is coming soon. If you want to keep track of the progress on this feature, [follow this issue](https://github.com/Galileo-Galilei/kedro-mlflow/issues/9).
+## What is metric tracking?
+
+Mlflow defines metrics as "Key-value metrics, where the value is numeric. Each metric can be updated throughout the course of the run (for example, to track how your model’s loss function is converging), and MLflow records and lets you visualize the metric’s full history".
+
+## How to version metrics in a kedro project?
+
+kedro-mlflow introduces a new ``AbstractDataSet`` called ``MlflowMetricsDataSet``. It is wrapper around dictionary with metrics which is returned by node and log metrics in MLflow.
+
+Since it is a ``AbstractDataSet``, it can be used with the YAML API. You can define it as:
+
+```yaml
+my_model_metrics:
+    type: kedro_mlflow.io.MlflowMetricsDataSet
+```
+
+## How to return metrics from node?
+
+Let assume that you have node which doesn't have any inputs and returns dictionary with metrics to log:
+
+```python
+def metrics_node() -> Dict[str, Union[float, List[float]]]:
+    return {
+        "metric1": 1.0,
+        "metric2": [1.0, 1.1],
+        "metric3": {"value": 1.1, "step": 1},
+        "metric4": [{"value": 1.1, "step": 1}, {"value": 1.2, "step": 2}]
+    }
+```
+
+As you can see above, ``kedro_mlflow.io.MlflowMetricsDataSet`` can take metrics as:
+ - ``float``,
+ - ``List[float]``
+ - ``Dict[str, key]``,
+ - ``List[Dict[str, key]]``
+
+To store metrics we need to define metrics dataset in Kedro Catalog:
+
+```yaml
+my_model_metrics:
+    type: kedro_mlflow.io.MlflowMetricsDataSet
+```
+
+Metrics stored in MLflow will be prefixed with dataset name. For our example it would be: ``my_model_metrics.metric1``, ``my_model_metrics.metric2``, ``my_model_metrics.metric3``, ``my_model_metrics.metric4``. 
```

Comment:
We should specify that it is the hook that "forces" the key to be the name in the catalog. Indeed it will only works if the pipeline is run with a `KedroContext` objec and not only with a `Pipeline` and `Runner` obejcts and it may be confusing

Reply:
:+1: